### PR TITLE
Create ui-c8y-1020-0-13-add-selector-to-every-component-to-get-rid-of-warning-6488.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by c8y-ui-automations[bot]

Ticket: [MTM-57343](https://cumulocity.atlassian.net/browse/MTM-57343)
Original PR: [6488](https://github.softwareag.com/IOTA/cumulocity-ui/pull/6488)

### CODE REVIEW

File: content/change-logs/application-enablement/ui-c8y-1020-0-13-add-selector-to-every-component-to-get-rid-of-warning-6488.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: add selector to every component to get rid of warning (#6488)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1020-0-13-add-selector-to-every-component-to-get-rid-of-warning-6488.md:3]: The Claude AI suggested to use the following title:
```suggestion
title: Added selector to every component to remove warnings
```

File: content/change-logs/application-enablement/ui-c8y-1020-0-13-add-selector-to-every-component-to-get-rid-of-warning-6488.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-57343
+version: 1020.0.13
+---
+add selector to every component to get rid of warning (#6488)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1020-0-13-add-selector-to-every-component-to-get-rid-of-warning-6488.md:17]: The Claude AI suggested to use the following body:
```suggestion
In an effort to improve code quality and maintainability, selectors have been added to every component in the codebase. Previously, the absence of selectors triggered warnings during development and testing. With this change, those warnings have now been eliminated, resulting in a cleaner and more maintainable codebase. This change does not impact the functionality or behavior of the components from a user perspective.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

**Review by rahultsag — APPROVED** [review]

**Comment by reey** [comment]
Closing as this was more or less just an internal change..

**Pull request closed without merging**